### PR TITLE
Require 2 or more phonics sounds

### LINKED ISSUE #85: Update Phonics Lessons in glen-world, glen-learn
- Phonics Lessons should require 2+ sounds
- Add 4th consonant sound (from Jason's code?)
- Save lesson as C3
- Make necessary changes on glen-learn (merge from Master)

### PR DESCRIPTION
**Pull request by psp2**
## Status:
:rocket: Ready

## Description :star2:

Fixes #85

### CODE REVIEW

File: src/pages/Confirmation/Confirmation.js [hunk 1]
```diff
@@ -18,10 +18,12 @@ function Confirmation(props) {
   const [handleWords, setWords] = useState(false);
   const [handleVocab, setVocab] = useState(false);
   const [needRowSpace, setNeedRowSpace] = useState(false);
+  // Phonics Lessons
+  const [typeCLessons] = useState(["C", "C1", "C2", "C3"]);
```
Comment (chyku) [inline src/pages/Confirmation/Confirmation.js]: make this a const in the utils folder (i think)! state variables are component-specific and usually meant to change

File: src/pages/CreateAssignment/CreateAssignment.js [hunk 2]
```diff
@@ -56,11 +58,11 @@ function CreateAssignment(props) {
 
     let deploymentNameMap = {};
     Promise.all(
-      deploymentAccountIds.map(id => {
+      deploymentAccountIds.map((id) => {
```
Comment (chyku) [inline src/pages/CreateAssignment/CreateAssignment.js]: was this prettier or something?

**Review by chyku — CHANGES_REQUESTED** [review]
one major thing & pls reformat bc i think most of these changes are prettier

File: src/pages/Confirmation/Confirmation.js (hunk 1 above)
Comment (psp2) [reply src/pages/Confirmation/Confirmation.js]: honestly I might just remove these (there's 1 or 2 other places I do this) and just put C3 in whenever I'm adding or checking the Phonics lesson type. The only reason I provided multiple C lesson types is I wasn't sure if we would need to support multiple types of phonics lessons down the line. Although based off of Sumit's email, we'd probably be fine just using C3

**Review by chyku — APPROVED** [review]
good stuff

**Pull request merged**